pkg/gitops/url_test.go

```diff
@@ -1,19 +1,12 @@
 package gitops_test
 
 import (
-	"testing"
-
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 
 	"github.com/weaveworks/eksctl/pkg/gitops"
-	"github.com/weaveworks/eksctl/pkg/testutils"
 )
 
-func TestSuite(t *testing.T) {
```

Comment:
Why remove this?

Reply:
See the second commit and onsi/ginkgo#704